src/main/authentication/authentication-config.json

```diff
@@ -1,7 +1,7 @@
 {
   "RESPONSE_TYPE": "code",
   "RESPONSE_MODE": "form_post",
-  "POLICY": "B2C_1A_SIGNUP_SIGNIN_PASSWORDLESS",
-  "IDENTITY_METADATA": "https://pib2csbox.b2clogin.com/pib2csbox.onmicrosoft.com/B2C_1A_SIGNUP_SIGNIN_PASSWORDLESS/v2.0/.well-known/openid-configuration",
+  "POLICY": "B2C_1_SignInUserFlow",
+  "IDENTITY_METADATA": "https://pib2csbox.b2clogin.com/pib2csbox.onmicrosoft.com/B2C_1_SignInUserFlow/v2.0/.well-known/openid-configuration",
```

Comment:
Should we capture the env. variable for the b2c screens?

Reply:
I can see this was requested to devOps on https://tools.hmcts.net/jira/browse/PUB-962

Reply:
Yep, I've only re-used what was already in the solution and asked Chris P to add those env variables in. Assuming we would need to add these for other 2 auth methods (Commpon Platform and HMCTS account)